src/utils/sessions.ts

```diff
@@ -0,0 +1,16 @@
+import { IncomingMessageWithSession, NextApiRequestWithSession, SessionContents } from '../interfaces';
```

Comment:
This file is already in implementation within another pr so once that is in develop this will get merged.